client/my-sites/site-settings/comment-markdown-toggle/index.jsx

```diff
@@ -0,0 +1,91 @@
+/**
+* External dependencies
+*/
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import ExternalLink from 'components/external-link';
+import InfoPopover from 'components/info-popover';
+import { isJetpackModuleActive, isActivatingJetpackModule } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { activateModule } from 'state/jetpack/modules/actions';
+
+class CommentDisplaySettings extends Component {
+	componentDidUpdate() {
+		const {
+			isActivatingMarkdownModule,
+			isMarkdownModuleActive,
+			fields,
+			siteId
+		} = this.props;
+
+		if ( isMarkdownModuleActive !== false ) {
+			return;
+		}
+
+		if ( ! fields.wpcom_publish_comments_with_markdown ) {
+			return;
+		}
+
+		if ( isActivatingMarkdownModule ) {
+			return;
+		}
+
+		this.props.activateModule( siteId, 'markdown', true );
+	}
+
+	isFormPending() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+		} = this.props;
+
+		return isRequestingSettings || isSavingSettings;
+	}
+
+	render() {
+		const {
+			fields,
+			handleToggle,
+			translate,
+		} = this.props;
+
+		return (
+			<div className="comment-markdown-toggle">
+				<div className="comment-markdown-toggle__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink href="http://en.support.wordpress.com/markdown-quick-reference/" target="_blank">
+							{ this.props.translate( 'Learn more about markdown' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Learn more about Markdown.' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).